js/mmoda.instrument.js

```diff
@@ -1124,22 +1112,31 @@ function panel_title(srcname, param) {
         // true);
 
         // Collect object name 
-        var allFormData = $("form#mmoda-name-resolve").serializeArray().map(function(item, index) {
+        var allFormData = $("form#mmoda-name-resolve").serializeArray().map(function(item) {
```

Comment:
Hm, these removals of `index` are not my changes, they somehow appeared as a result of merging. I'm not sure if this is something critical, it's not used inside...

Reply:
I confirm they are from the latest changes @motame introduced, I also had to merge those changes today